web/styles/modal.css

```diff
@@ -157,6 +157,22 @@
     }
 }
 
+.save-success {
```

Comment:
I am not sure of the selector here but I guess we can just make this more specific for now by adding `#user-profile-modal` and also because we have this CSS here. We usually place the CSS to be used across the app in `app_components.css`.

Reply:
Done.